src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -43,14 +43,13 @@ def load_arguments(self, _):
         c.argument('service_runtime_network_resource_group', options_list=['--service-runtime-network-resource-group', '--svc-nrg'], help='The resource group where all network resources for Azure Spring Cloud service runtime will be created in.', validator=validate_node_resource_group)
         c.argument('app_network_resource_group', options_list=['--app-network-resource-group', '--app-nrg'], help='The resource group where all network resources for apps will be created in.', validator=validate_node_resource_group)
         c.argument('enable_java_agent', is_preview=True, arg_type=get_three_state_flag(), help="Enable java in-process agent", validator=validate_java_agent_parameters)
+        c.argument('app_insights_key', help="If enable-java-agent is set to true, support both connection string and instrumentation key of the existed Application Insights. Otherwise, only instrumentation key is supported.", validator=validate_tracing_parameters)
     with self.argument_context('spring-cloud update') as c:
+        c.argument('app_insights_key', help="Instrumentation key of the existing Application Insights", validator=validate_tracing_parameters)
         c.argument('sku', type=str, validator=validate_sku, help='Name of SKU, the value is "Basic" or "Standard"')
```

Comment:
Why do we have different help infos for the same parameter `app_insights_key`?

Reply:
Sry, It's outdate. And not able to find the context.
For lastest case, 
`app_insights_key` under `az spring-cloud update` is be marked as deprecated, and will be removed after migration phase
where as `app_insights_key` under `az spring-cloud create` will be long supported. So they has different description.